modules/oe_content_news/oe_content_news.info.yml

```diff
@@ -0,0 +1,31 @@
+name: OpenEuropa News Content
+description: Contains the EC corporate News content type.
+package: OpenEuropa
+
+type: module
+core: 8.x
+dependencies:
+  - oe_content:oe_content
+  - oe_media:oe_media
+  - oe_media_avportal:oe_media_avportal
+
+config_devel:
+  - core.entity_form_display.node.oe_news.default
+  - core.entity_view_display.node.oe_news.default
+  - core.entity_view_display.node.oe_news.teaser
```

Comment:
I missed it in the page ticket, but let's remove the teaser display as we don't have the specifications for it nor any use case at the moment.